src/test/java/net/mcreator/integration/generator/GTModElements.java

```diff
@@ -102,6 +113,16 @@ public static void runTest(Logger LOG, String generatorName, Random random, Work
 				// testing if all element files were properly deleted
 				modElementFiles = workspace.getGenerator().getModElementGeneratorTemplatesList(modElement).stream()

```

Comment:
getModElementGeneratorTemplatesList should include list templates too, this function by definition returns list of all templates for said mod element, be it templates or not.

Reply:
The problem is that for UI cases list templates should be separated from usual templates to get grouped. Would a `GeneratorTemplatesList.removeListTemplates(List<GeneratorTemplate>)` method be an option?

Reply:
GeneratorTemplate could have type field and then you can sort by type in the UI when needed.

But the logic in the generator system should work the way it does now, getModElementGeneratorTemplatesList should return list of all templates

Reply:
✔️ Fixed